src/MongoDB/Manager.c

```diff
@@ -337,6 +350,7 @@ static PHP_METHOD(Manager, executeCommand)
 	bool                  free_options    = false;
 	zval*                 zreadPreference = NULL;
 	uint32_t              server_id       = 0;
+	zval*                 zsession        = NULL;
```

Comment:
Minor nit, but I'd move this above the `server_id` declaration since it'd used first. Also keeps two `zval*` declarations side-by-side, which looks nice :)